DfE.FindInformationAcademiesTrusts.Data.Hardcoded/ExploreEducationStatisticsPhaseType.cs

```diff
@@ -0,0 +1,11 @@
+namespace DfE.FindInformationAcademiesTrusts.Data.Hardcoded;
+
+public enum ExploreEducationStatisticsPhaseType
+{
+    // These types are taken from https://explore-education-statistics.service.gov.uk/data-tables/permalink/25bc8d0b-c700-4000-1b8a-08dbb99e3fd8
```

Comment:
These comments are good, but they would be better as a `///` above the enum declaration - this will allow intellisense to pick them up